### ISSUE TITLE
Cool project!

### ISSUE THREAD

**Issue by uestccokey** [Geometrie/LZRobe#1]
It will be better if there are some screenshots.

**Comment by bvandenbon** [comment]
Also interested in screenshots, wondering if it is similar to the project I run on https://www.zbaduk.com

**Comment by Geometrie** [comment]
> It will be better if there are some screenshots.

I tried to use double buffer to reduce the blinks while analyzing the game record.

**Comment by Geometrie** [comment]
> Also interested in screenshots, wondering if it is similar to the project I run on https://www.zbaduk.com

Yes, it is similar. I've learned a lot from those projects. Thanks for your recommendation.

**Comment by l1t1** [comment]
you could get some idea from other software, such as ana, goreviewpartner.

**Comment by Geometrie** [comment]
> you could get some idea from other software, such as ana, goreviewpartner.

Thanks for your advice!
